scripts/check-diff-line-width.sh

```diff
@@ -26,6 +26,7 @@ ERROR_THRESHOLD=140
 git --no-pager diff $BASE_HASH -U0 -- cdc pkg cmd \
 	-- ':(exclude)*_gen.go' \
 	-- ':(exclude)*_gen_test.go' \
+	-- ':(exclude)*_test.go' \
```

Comment:
It is better to use raw strings in the test and delete this line.

Reply:
ok